setup.cfg

```diff
@@ -1,72 +1,8 @@
-[metadata]
-name = goodconf
-version = 2.0.2.dev0
-description = Load configuration variables from a file or environment
-long_description = file: README.rst
-author = Peter Baumgartner
-author_email = pete@lincolnloop.com
-url = https://github.com/lincolnloop/goodconf/
-keywords = env, config, json, yaml
-license = MIT
-classifiers =
-    Development Status :: 5 - Production/Stable
-    Intended Audience :: Developers
-    License :: OSI Approved :: MIT License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-
-[options]
-packages = find:
```

Comment:
The find stuff is no longer needed, setuptools & setuptools_scm seems to do the correct thing.